app/src/ui/app-error.tsx

```diff
@@ -132,6 +125,14 @@ export class AppError extends React.Component<IAppErrorProps, IAppErrorState> {
     return <p className={className}>{e.message}</p>
   }
 
+  private getTitle(error: Error) {
+    if (isErrorWithMetaData(error) && isCloneError(error)) {
```

Comment:
Super-small nit: I don't think we need to check whether the error has metadata here (since `isCloneError()` already does:

```suggestion
    if (isCloneError(error)) {
```

Reply:
Good catch!